imagetest/test_suites/image_validation/cloud_test.go

```diff
@@ -0,0 +1,97 @@
+package imagevalidation
+
+import (
+	"io/ioutil"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+const (
+	chronyService = "chronyd"
+	ntpdService   = "ntp"
+)
+
+var ntpConfig = []string{"/etc/ntp.conf"}
+var chronyConfig = []string{"/etc/chrony.conf", "/etc/chrony/chrony.conf", "/etc/chrony.d/gce.conf"}
+
+// TestNTPService Verify that ntp package exist and configuration is correct.
+// For SUSE, the ntp service is ntpd. For others, the ntp service is chronyd
+func TestNTPService(t *testing.T) {
+	image, err := utils.GetMetadata("image")
+	if err != nil {
+		t.Fatalf("Couldn't get image from metadata")
+	}
+	var servicename string
+	var configPaths []string
+	switch {
+	case strings.Contains(image, "debian-10"):
+	case strings.Contains(image, "debian-11"):
+	case strings.Contains(image, "rhel"):
+	case strings.Contains(image, "centos"):
+	case strings.Contains(image, "sles"):
+		servicename = chronyService
+		configPaths = chronyConfig
+	case strings.Contains(image, "ubuntu"):
+		switch {
+		case strings.Contains(image, "ubuntu-1604"):
+		case strings.Contains(image, "ubuntu-minimal-1604"):
+			servicename = ntpdService
+			configPaths = ntpConfig
+		default:
+			servicename = chronyService
+			configPaths = chronyConfig
+		}
+	default:
+		servicename = ntpdService
+		configPaths = ntpConfig
+	}
+
+	configContent, err := readNTPConfig(configPaths)
+	if err != nil {
+		t.Fatalf("failed reading ntp config file %s", err)
+	}
+	ntpConfigs := strings.Split(configContent, "\n")
```

Comment:
np it seems you could just have readNTPConfig return []string if that's what's useful to you

Reply:
Move into readNTPConfig function